deltacat/catalog/model/table_definition.py

```diff
@@ -38,3 +43,12 @@ def stream(self) -> Stream:
     @property
     def native_object(self) -> Optional[Any]:
         return self.get("nativeObject")
+
+    @property
+    def scan_planner(self) -> Optional[ScanPlanner]:
+        return self.get("scan_planner")
+
+    def create_scan_plan(self, pushdown: Optional[Pushdown] = None) -> ScanPlan:
+        if not self.scan_planner:
+            raise RuntimeError(f"ScanPlanner is not initialized for table '{self.table.table_name}' of namespace '{self.table.namespace}'")
+        return self.scan_planner.create_scan_plan(table_name=self.table.table_name, namespace=self.table.namespace, pushdown=pushdown)
```

Comment:
nit/codestyle: Try to keep line lengths to 80 characters (in accordance with PEP8: https://peps.python.org/pep-0008/... primary reasons IMO are to assist with code readability of large text for individuals who may struggle to read small text, and to allow side-by-side vertical viewing of 2 codefiles on a single monitor for those that have no problem reading smaller text). 

Also prefer to separate 2+ input args on multiple lines to aid with diff resolution on subsequent PRs. (i.e., to facilitate fast visual resolution of which params were updated/added/removed in a multi-param function invocation or delcaration in subsequent PRs). Note that you'll need to end the argument list with a trailing comma to prevent Black from auto-formatting this back down to a single line.

Reply:
thanks for providing the context - I was able to format the code by calling `make lint` which I missed earlier